### DISCUSSION TITLE
How to Solve AWS SES Email Sending Issue with Vercel Deployment?

### DISCUSSION THREAD

**Discussion by rhadooqoo** [vercel/next.js#64788] [Help]
### Summary

I'm encountering an issue: I'm unable to send emails using AWS SES. It functions only on localhost or after deploying it on Vercel, but that's only with hardcoded credentials. Somehow, Vercel isn't recognizing the credentials I've set in its environment variables. It used to function properly, but now it seems to have ceased working altogether.

It seems that if I make the variables public using NEXT_PUBLIC_ in the name, it works! Can someone help me in this matter, please?

**Comment by v-stickykeys** [comment]
I had this issue and the problem was I forgot to pass credentials directly to the SES client like
```
export const client = new SESClient({
  region: REGION,
  credentials: {
    accessKeyId: process.env.CLOUD_ACCESS_KEY_ID || "",
    secretAccessKey: process.env.CLOUD_SECRET_ACCESS_KEY || "",
  }
});
```
